app/serializers/rest/account_serializer.rb

```diff
@@ -55,6 +55,10 @@ def header_static
     full_asset_url(object.suspended? ? object.header.default_url : object.header_static_url)
   end
 
+  def created_at
+    object.created_at.midnight.iso8601
+  end
+
```

Comment:
Why not do like `last_status_at` and use `to_date` instead of including midnight time?

Reply:
Well, in this API it was already datetime, so that might count as a breaking change... probably not a big deal though, I’m not sure

Reply:
Alright, fine by me, considering whether changing that to a date would be a breaking change can be postponed, using midnight time is safe.